db/models.py

```diff
@@ -40,13 +53,78 @@
 
 
 class MovieSession(models.Model):
-    show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(
+    show_time: models.DateTimeField = models.DateTimeField()
```

Comment:
Type annotations for Django model fields are not necessary and could lead to confusion. It's better to use Django's field types directly without type annotations.